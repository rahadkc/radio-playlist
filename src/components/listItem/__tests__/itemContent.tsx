import { cleanup, render, screen } from '@testing-library/react'
import ItemContent from '../itemContent'

const name = 'Test FM'
const value = '103.8'
const number = 98.4

describe('<ItemContent> component', () => {
  afterEach(cleanup)

  test('render ListItem component', () => {
    render(<ItemContent style={{}} name={name} value={value} />)
    expect(screen.queryByText(name)).toBeInTheDocument()
    expect(screen.queryByText(value)).toBeInTheDocument()

    render(<ItemContent style={{}} name={name} value={number} />)
    expect(screen.queryByText(number)).toBeInTheDocument()
    expect(screen.queryByText(number)).toMatchSnapshot()
  })
})
