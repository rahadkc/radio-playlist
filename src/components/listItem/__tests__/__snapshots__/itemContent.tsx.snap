// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ItemContent> component render ListItem component 1`] = `
<div
  class="value"
>
  98.4
</div>
`;
